src/apps/ReleaseControl/components/sidesheet/ReleaseControlSidesheet/EditTabs/EditButtonBar.tsx

```diff
@@ -0,0 +1,62 @@
+import { Button, Progress } from '@equinor/eds-core-react';
+import { disableEditMode } from '../../../../Atoms/editModeAtom';
+import { DRCFormAtomApi } from '../../../../Atoms/formAtomApi';
+import { useReleaseControlContext } from '../../../../hooks/useReleaseControlContext';
+import { useReleaseControlMutation } from '../../../../hooks/useReleaseControlMutation';
+import { useRequestMutations } from '../../../../hooks/useRequestMutations';
```

Comment:
```suggestion
import { useReleaseControlContext, 
useReleaseControlMutation,
 useRequestMutations } from '../../../../hooks/';
```

can you create an index.ts in the hooks folder with all the exports